bigpicture.md

```diff
@@ -0,0 +1,103 @@
+# BigPicture-SE-user-docs
+
+This repository contains user information on how to submit files to the Swedish
+Big Picture node. This document serves as an outline of the submission process, and links to more detailed guides on how to perform the individual steps for creating a submission.
+
+Currently there exist two ways to perform the submission process to the Swedish Big Picture node. 
+
+# The automated way
+
+The first and recommended one,uses the sda-cli tool developed by the Swedish Big Picture team. This solution can be used in any system with internet access and only requires downloading a binary file from [here](https://github.com/NBISweden/sda-cli/releases). This way is recommended for regular users, as well as for systems where the user lacks installation rights. Detailed instructions can be found [here](https://github.com/NBISweden/sda-cli/).
+
+# The manual way
+
+Alternatively, systems administrators and/or people who want to have more control over the data submission process can follow the manual steps. This requires the installation of two tools and the detailed instructions can be found in the following sections.
+
+## Preparing the system
+
+Submitting data requires two tools, [crypt4gh](https://www.ga4gh.org/news/crypt4gh-a-secure-method-for-sharing-human-genetic-data/) for encrypting the data, and [s3cmd](https://s3tools.org/s3cmd) for uploading to the Swedish Big Picture node.
+
+To prepare the system follow the instructions [here](binaries.md). This is a **required step**
+in order to setup your system with the required software.
+
+## Encrypting
+
+The first step of a submission is to encrypt the submission data files with the
+`crypt4gh` encryption tool. This can be limited to these two or three steps:
+
+ - Download the crypt4gh public key
+
+   For the Swedish Big Picture node to be able to understand the encrypted files when they are uploaded, they need to be encrypted with the correct public key. This key can be downloaded from this repository with this command:
+   ```bash
+   wget https://raw.githubusercontent.com/NBISweden/EGA-SE-user-docs/main/crypt4gh_bp_key.pub
+   ```
+
+ - (Optional) Create a personal key pair
+
+   The creation of a key pair is very simple using crypt4gh. Run the following command, replacing [my-key] with the name of the key (can be anything) and specifying a passphrase when requested:
+   
+   ```bash
+   crypt4gh generate --name=[my-key]
+   ```
+
+   To verify that the key pair was created, run the `ls` command and make sure the keys you specified exist in the folder.
```

Comment:
`ls` is not in DOS. Would it make sense if we avoid references to os specific commands?